tests/monotouch-test/AVFoundation/AVCapturePhotoBracketSettingsTest.cs

```diff
@@ -0,0 +1,29 @@
+#if XAMCORE_2_0 && !__WATCHOS__ && !__TVOS__
+
+using System;
+using Foundation;
+using AVFoundation;
+using NUnit.Framework;
+
+namespace monotouchtest {
+	[TestFixture]
+	[Preserve (AllMembers = true)]
+	public class AVCapturePhotoBracketSettingsTest {
+
+		[Test]
+		public void TestConstructor ()
+		{
```

Comment:
that will need a version check so it's only executed on Xcode8+ 
otherwise it will fail when QA test this with older devices